ROMFS/tap_common/init.d/rcS

```diff
@@ -145,97 +151,58 @@ then
 	fi
 fi
 
-gps start -d /dev/ttyS0
-
-# waypoint storage
-# REBOOTWORK this needs to start in parallel
-if dataman start $DATAMAN_OPT
-then
-fi
-
-#
-# Sensors System (start before Commander so Preflight checks are properly run)
-#
-sh /etc/init.d/rc.sensors
-
-commander start
-
-#
-# Start CPU load monitor
-#
-load_mon start
-
-# Start MAVLink on the gimbal port
-#mavlink start -r 1200 -d /dev/ttyS1
-
-# Start MAVLink on USB, developers can use the MAVLink shell
-mavlink start -r 60000 -d /dev/ttyACM0 -m config
-
 #
-# Logging
+# Start the RC input driver
```

Comment:
fmu is not used for RC input anymore.
There should be a separate `rc_input start`

Reply:
Done. Thanks @bkueng .